internal/ui/tailwind/tailwind.go

```diff
@@ -0,0 +1,33 @@
+package tailwind
```

Comment:
We don't need package for tailwind. We can just keep it in the `ui` package.

Reply:
The file can still be called `tailwind.go` when moved to the `ui` package. You can rename the functions to:
- `tailwindBuild` or `buildTailwind`
- `tailwindWatch` or `watchTailwind`